src/Polly.RateLimiting/RateLimiterRejectedException.cs

```diff
@@ -74,6 +116,11 @@ public RateLimiterRejectedException(string message, TimeSpan retryAfter, Excepti
     /// </remarks>
     public TimeSpan? RetryAfter { get; }
 
+    /// <summary>
+    /// Gets the source of the strategy which has thrown the exception, if known.
+    /// </summary>
+    public string? TelemetrySource { get; }
```

Comment:
Why not use telemetry source here? The strategy name is not enough to uniquely identify the strategy.

Reply:
See https://github.com/App-vNext/Polly/pull/2346#discussion_r1803160193

Reply:
Thanks, added a comment :)